airbyte-integrations/connectors/source-google-ads/acceptance-test-config.yml

```diff
@@ -20,39 +19,53 @@ acceptance_tests:
       - config_path: "secrets/config.json"
         expect_records:
           path: "integration_tests/expected_records.jsonl"
-        timeout_seconds: 600
+        timeout_seconds: 3600
         empty_streams:
-          - name: "display_topics_performance_report"
-            bypass_reason: "Stream not filled yet."
           - name: "account_labels"
-            bypass_reason: "Unable to seed the stream"
+            bypass_reason: "Unsufficient permissions"
+          - name: "unhappytable"
+            bypass_reason: "Empty testing stream"
+          - name: "shopping_performance_report"
+            bypass_reason: "No shopping campaign, need item for sale "
+          - name: "display_topics_performance_report"
+            bypass_reason: "No data for this date range, tested in next config"
           - name: "click_view"
-            bypass_reason: "Stream not filled yet."
+            bypass_reason: "Stream has data only for last 90 days, next config is used for testing it"
+        ignored_fields:
+          accounts:
+            - name: customer.optimization_score_weight
+              bypass_reason: "value can be updated by Google Ads"
+            - name: customer.optimization_score
+              bypass_reason: "value can be updated by Google Ads"
+      - config_path: "secrets/config_click_view.json"
+        expect_records:
+          path: "integration_tests/expected_records_click.jsonl"
+        timeout_seconds: 3600
+        empty_streams:
+          - name: "account_labels"
+            bypass_reason: "Unsufficient permissions"
           - name: "unhappytable"
-            bypass_reason: "Stream not filled yet."
+            bypass_reason: "Empty testing stream"
```

Comment:
can we populate this stream with data?

Reply:
Purposely empty stream; that's why it is called unhappytable

Reply:
Stream was deleted